website/docs/r/vapp_vm.html.markdown

```diff
@@ -333,6 +334,8 @@ example for usage details. **Deprecates**: `network_name`, `ip`, `vapp_network_n
   * `ip_allocation_mode=MANUAL` - **`ip`** value must be valid IP address from a subnet defined in `static pool` for network.
 
   * `ip_allocation_mode=NONE` - **`ip`** field can be omitted or set to an empty string "". Empty string may be useful when doing HCL variable interpolation.
+  
+  * `connected` - It defines if NIC is connected or not.
```

Comment:
Can you just briefly mention the situation for NICs with NoneNetwork - the thing that they cannot be connected by default.

Reply:
added